tests/integration/devfile/cmd_devfile_watch_test.go

```diff
@@ -9,13 +9,20 @@ import (
 	. "github.com/onsi/gomega"
 
 	"github.com/openshift/odo/tests/helper"
+	"github.com/openshift/odo/tests/integration/devfile/utils"
 )
 
 var _ = Describe("odo devfile watch command tests", func() {
 	var namespace string
 	var context string
+	var cmpName string
+	var projectDirPath string
 	var currentWorkingDirectory string
 
+	// TODO: all oc commands in all devfile related test should get replaced by kubectl
+	// TODO: to goal is not to use "oc"
+	oc := helper.NewOcRunner("oc")
```

Comment:
Done for `oc` only.

The `CliRunner` interface will not work for `dockerRunner` as of today, since most of those extracted `CliRunner` functions are not implemented for `dockerRunner`. Opened an issue to clean this up - https://github.com/openshift/odo/issues/3225